### ISSUE TITLE
"client serialize-transfer-sui" and "keytool sign" don't work together on devnet branch

### ISSUE THREAD

**Issue by martin-chainflip** [MystenLabs/sui#7473]
## Steps to Reproduce Issue
```
➜ cargo install --locked --git https://github.com/MystenLabs/sui.git --branch devnet sui
➜ sui genesis
➜ sui start
```
in another terminal
```
➜ sui -V
sui 0.21.1
➜ sui client active-address
0xe7e3f21b76d7299a2412e19e2f4b3843b7dd8dc2
➜ sui client objects
... // Pick the first one
0x40942dcaa26bfa9d64524d6917bc1f3917f6d8ff
...
➜ sui client serialize-transfer-sui --to 0xe7e3f21b76d7299a2412e19e2f4b3843b7dd8dc2 --sui-coin-object-id 0x40942dcaa26bfa9d64524d6917bc1f3917f6d8ff --gas-budget 1000
Data to sign: AAAAAAPn4/IbdtcpmiQS4Z4vSzhDt92NwgDn4/IbdtcpmiQS4Z4vSzhDt92NwkCULcqia/qdZFJNaRe8HzkX9tj/AQAAAAAAAAAgi4VqSk76TcnK9OMtVMgGK7TXo6o3dsYbiRFc8llo1l8BAAAAAAAAAOgDAAAAAAAA
➜ sui keytool sign --address 0xe7e3f21b76d7299a2412e19e2f4b3843b7dd8dc2 --data AAAAAAPn4/IbdtcpmiQS4Z4vSzhDt92NwgDn4/IbdtcpmiQS4Z4vSzhDt92NwkCULcqia/qdZFJNaRe8HzkX9tj/AQAAAAAAAAAgi4VqSk76TcnK9OMtVMgGK7TXo6o3dsYbiRFc8llo1l8BAAAAAAAAAOgDAAAAAAAA
...
unexpected end of input
```

## Expected Result
I would expect the command to produce a signature for the transaction. If I use the main branch (sui -V reports 0.23.0) it works as expected.

## System Information

* OS: Apple M1 Max with MacOS Ventura 13.1
* Compiler: cargo 1.68.0-nightly (8c460b223 2023-01-04), rustc 1.68.0-nightly (659e169d3 2023-01-04)

**Comment by danaiballa** [comment]
This problem cannot be reproduced on current sui devnet version (0.23.0) @666lcz, so maybe this issue can be closed.

**Comment by martin-chainflip** [comment]
I can confirm that this is fixed since "devnet" was updated to version 0.23. Thanks
